### DISCUSSION TITLE
Deploy Stucked in Queue and Triger Not Started after push to Master

### DISCUSSION THREAD

**Discussion by Helpoo-App** [codemagic-ci-cd/codemagic-docs#2226] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. I commit my Changes 
2. Push to Master 
3. Deploy Not Notified on slack , or on Platform Dashboard
4. Marked As Queued 

### Expected results

I excepted to be started  

### Actual results

Queued 

### Build id (optional)

65661a643feadb14d209a0be

**Comment by mikhail-tokarev** [comment]
there was a some delay, but it should be ok now. Let me know if the issue still persists for you

**Reply by Helpoo-App** [reply]
Thanks for you Quick Reply i think all is good Now 
and really i Appreciated your effort and expected to increase the Time of Build :D as a compensation :D
